lib/resources/majors.tsx

```diff
@@ -2395,135 +2395,199 @@ const bsBME: Major = {
+        },
+        {
+          description:
+            '<b>Fundamentals of Health Policy & Management</b> <br /> AS.280.340 Fundamentals of Health Policy & Management (Spring)',
+          required_credits: 3,
+          criteria: 'AS.280.340[C]',
+        },
+        {
+          description:
+            '<b>Public Health Biostatistics</b> <br /> AS.280.345 Public Health Biostatistics (Fall) <br /> <i>OR</i> <br /> Other statistics courses may apply with advisor approval.',
+          required_credits: 3,
+          criteria: 'AS.280.345[C]',
+        },
+        {
+          description:
+            '<b>Fundamentals of Epidemiology</b> <br /> AS.280.350 Fundamentals of Epidemiology (Fall & Spring)',
+          required_credits: 3,
+          criteria: 'AS.280.350[C]',
+        },
+      ],
+    },
+    {
+      name: 'Intermediate Public Health Courses at Homewood',
+      required_credits: 12,
+      min_credits_per_course: 3,
+      double_count: ['All'],
+      description:
+        'One course at the 200-400 level focusing on social and behavioral aspects of public health from the list below.' +
+        'Please see your PHS advisor for a list of current courses that will satisfy the requirement.',
+      criteria:
+        'AS.230.341[C]^OR^AS.280.355[C]^OR^((AS Public Health Studies[D])^AND^(200[L]^OR^300[L]^OR^400[L]))',
+      fine_requirements: [
+        {
+          description:
+            '<b>Social and Behavioral Course</b> <br /> AS.230.341 Sociology of Health and Illness (other options may be approved by your advisor) <br /> <i>OR</i> <br />AS.280.355 Introduction to Social and Behavioral Determinants of Health',
+          required_credits: 3,
+          criteria: 'AS.230.341[C]^OR^AS.280.355[C]',
+        },
+        {
+          description:
+            'Three additional Public Health courses at the 200-400 level',
+          required_credits: 9,
+          criteria:
+            '(AS Public Health Studies[D])^AND^(200[L]^OR^300[L]^OR^400[L])',
+        },
+      ],
+    },
+    {
+      name: 'Applied Experience',
+      required_credits: 1,
+      min_credits_per_course: 1,
+      double_count: ['All'],
+      description:
+        'The Applied Experience (AE) is a required experiential learning component of the Public Health Studies' +
+        ' curriculum. The purpose of the AE is to provide students with supervised, hands-on experience in a professional' +
+        ' public health setting. The Applied Experience gives students an opportunity to explore an area of interest within' +
+        ' the field of public health by actively engaging in and directly contributing to a public health project or program.' +
+        ' The AE must be at least 80 hours and 4 weeks long under the supervision of a public health profession.',
+      criteria: 'AS.280.500[C]',
+    },
+    {
+      name: 'Courses at Johns Hopkins Bloomberg School of Public Health (JHSPH)',
+      required_credits: 10,
```

Comment:
you make a good poitn about how this will never be satisfied / uCredit doesn't have these courses. That is because we cache courses from KSAS/WSE but not Bloomberg school of public health. We have two choices, 

1. start caching bloomberg courses 
2. just make this distribution always satisfied

Reply:
number 2 is preferred hehe